test/app-tap/datetime.test.lua

```diff
@@ -1740,11 +1740,11 @@ test:test("Time :set{} operations", function(test)
             'day 9')
     test:is(tostring(ts:set{ hour = 6 }),  '2020-11-09T06:31:11+0300',
```

Comment:
That's the most epic. We have set manually to 6 hours and, regardless of that, we were expecting 4h on return. And it was not confusing us at any case.